msg/vehicle_angular_velocity.msg

```diff
@@ -4,4 +4,6 @@ uint64 timestamp_sample # timestamp of the data sample on which this message is
 
 float32[3] xyz		# Bias corrected angular velocity about the FRD body frame XYZ-axis in rad/s
 
+uint16 dynamic_range    # dynamic range of the gyro (degs/s)
```

Comment:
```suggestion
float32 dynamic_range    # dynamic range of the gyro (rad/s)
```

In this context should would call it something other than dynamic_range? Maybe maximum_angular_velocity or similar?

Reply:
I just did a short survey and we could follow the manufacturers' datasheet nomenclature... "scale range", "dynamic range", "range", "sensor range"... we just need to pick what works for PX4.  I'm good with whatever!